cmd/agent/subcommands/diagnose/command.go

```diff
@@ -86,11 +84,10 @@ func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 		Short: "Validate Agent installation, configuration and environment",
 		Long:  ``,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			utillog.SetupLogger(seelog.Disabled, "off")
```

Comment:
This line had the same purpose as https://github.com/DataDog/datadog-agent/pull/26923/files#diff-009da02793a5726889945bdb1cbcd0362a83aa0f770cc8d091c23508b27be955L94
```
LogParams: logimpl.ForOneShot("CORE", "off", true),
```
but did not allowed logs to be written during configuration initialization if the log level was overridden by provided configuration.